File: Soal-4/PRAK304-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int main() {
    int bilangan;

    scanf("%d", &bilangan);

    if (bilangan == 0) {
        printf("nol");
    }
    else if (bilangan > 0 && bilangan < 10) {
        printf("satuan");
    }
    else if (bilangan > 10 && bilangan < 20) {
        printf("belasan");
    }
    else if (bilangan >= 20 && bilangan <= 99 || bilangan == 10) {
        printf("puluhan");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}

File: Soal-4/PRAK304-2410817310014-INDRA_SURYADILAGA.py
bilangan = int(input())

if bilangan == 0 :
  print("nol")
elif bilangan > 0 and bilangan < 10 :
  print("satuan")
elif bilangan > 10 and bilangan <= 19 :
  print("belasan")
elif bilangan >= 20 and bilangan <= 99 or bilangan == 10:
  print("puluhan")
else :
  print("Anda Menginput Melebihi Limit Bilangan")